SonarQube.TeamBuild.PreProcessor/WebClientDownloader.cs

```diff
@@ -21,8 +21,13 @@ public WebClientDownloader(string username, string password)
             // SONARMSBRU-169 Support TLS versions 1.0, 1.1 and 1.2
             ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls;
 
+            if (password == null)
+            {
+                password = "";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Introduce a new variable instead of reusing the parameter "password". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS1226)